tests/vm_nesting.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+snap_channel="latest/edge"
+
+function parallel() {
+        seq 1 $1 | xargs -P $1 -I "{}" "${@:2}"
+}
+
+function init() {
+	parallel $1 lxc init ubuntu:jammy "t{}" $2
+}
+
+function conf() {
+	parallel $1 lxc config set "t{}" $2
+}
+
+function device_add() {
+	parallel $1 lxc config device add "t{}" $2 $3 $4
+}
+
+function start() {
+	args=""
+
+	for i in $(seq 1 $1); do
+		args="t$i $args"
+	done
+
+	echo "Start $args"
+	lxc start $args
+}
+
+function wait() {
+	parallel $1 bash -c "while true; do if lxc shell t{}; then break; fi; sleep 1; done"
+}
+
+function copy() {
+	parallel $1 lxc file push $2 "t{}$3"
+}
+
+function cmd() {
+	parallel $1 lxc exec "t{}" -- bash -c "$2"
+}
+
+function delete() {
+	args=""
+
+	for i in $(seq 1 $1); do
+                args="t$i $args"
+        done
+
+	echo "Delete $args"
+	lxc delete -f $args
+}
+
+if [[ ! -f $1 ]]
+then
+	echo "Specify the path to the LXD binary for sideloading"
+	exit 1
+fi
+
+if [[ ! $(which lxc) ]]
```

Comment:
Please use `command -v` instead of `which`.

Reply:
Didn't know about `command`, changed.

Reply:
It's a shell builtin and at one point `which` in `debianutils` was deprecated and complaining about it. So `command -v` is faster and should be trouble free :)

Reply:
Thanks for the insight, I am all in for stability :)